Build process for fresh XNA solution fails (problem with pipeline in ContentProject?)

I'm trying to port a mini game I made for browsers. These are my first steps with XNA and JSIL so I tried to start with baby steps (and failed)

**Preconditions**
1.  Installed XNA (XNA 4.0 Refresh - https://mxa.codeplex.com/releases) because the XNA Installer from Microsoft I found via Google didn't work
2. Setup a fresh XNA Solution (automatically creates two projects - game and content)
3. Build and Start in Visual Studio 2015 - works like a charm
4. git cloned JSIL (including all submodules) + Build All _(JSIL.Profiles.XNA4.dll is present)_

**Compile with JSILc.exe**
`jsilc.exe -o "D:\Code\_Anon_\XNA_JSIL_Test\web" "D:\Code\_Anon_\XNA_JSIL_Test\XNA_JSIL_Test.sln"`

**_Output_**

```
// Starting out-of-process solution build with ID 'JSIL.Build49C5C6A0'...
// Running target 'Build' of 'D:\Code\_Anon_\XNA_JSIL_Test\XNA_JSIL_Test.sln' ...
// Generating MSBuild projects for solution 'XNA_JSIL_Test.sln'... 1 project(s) generated.
// Building project 'D:\Code\_Anon_\XNA_JSIL_Test\XNA_JSIL_Test.sln.metaproj'...
__________________________________________________
Projekt "D:\Code\_Anon_\XNA_JSIL_Test\XNA_JSIL_Test.sln.metaproj", Build Ziel(e):

__________________________________________________
Das Projekt "D:\Code\_Anon_\XNA_JSIL_Test\XNA_JSIL_Test.sln.metaproj" erstellt "D:\Code\_Anon_\XNA_JSIL_Test\XNA_JSIL_Test\XNA_JSIL_Test\XNA_JSIL_Test.csproj" (Standardziele):

__________________________________________________
Das Projekt "D:\Code\_Anon_\XNA_JSIL_Test\XNA_JSIL_Test\XNA_JSIL_Test\XNA_JSIL_Test.csproj" erstellt "D:\Code\_Anon_\XNA_JSIL_Test\XNA_JSIL_Test\XNA_JSIL_TestContent\XNA_JSIL_TestContent.contentpr
oj" (Standardziele):

C:\Program Files (x86)\MSBuild\Microsoft\XNA Game Studio\v4.0\Microsoft.Xna.GameStudio.ContentPipeline.targets(78,5): error MSB4062: Die BuildContent-Aufgabe konnte nicht aus der Microsoft.Xna.Framewo
rk.Content.Pipeline, Version=4.0.0.0, Culture=neutral, PublicKeyToken=842cf8be1de50553-Assembly geladen werden. Die Datei oder Assembly "Microsoft.Xna.Framework.Content.Pipeline, Version=4.0.0.0, Cult
ure=neutral, PublicKeyToken=842cf8be1de50553" oder eine Abhängigkeit davon wurde nicht gefunden. Das System kann die angegebene Datei nicht finden. Stellen Sie sicher, dass die <UsingTask>-Deklaration
 korrekt ist, die Assembly und alle zugehörigen Abhängigkeiten verfügbar sind und die Aufgabe eine öffentliche Klasse enthält, die Microsoft.Build.Framework.ITask implementiert.
Die Erstellung des Projekts "XNA_JSIL_TestContent.contentproj" ist abgeschlossen -- FEHLER.
// Compilation failed for target 'Build': Unknown error
// Out-of-process solution build completed successfully.
// Auto-selected the profile 'XNA4' for this project.
// ContentOutputDirectory is not set. Skipping XNA content processing.
// No assemblies specified to translate. Exiting.
```

Would be great if somebody can give me a hint.
_Thanks in advance_

--- a/Compiler/SolutionBuilder.cs
+++ b/Compiler/SolutionBuilder.cs
@@ -159,7 +159,7 @@ namespace JSIL.SolutionBuilder {
                 pipeId, PipeDirection.InOut, 1, PipeTransmissionMode.Message, PipeOptions.Asynchronous
             )) {
                 var psi = new ProcessStartInfo {
-                    FileName = JSIL.Internal.Util.GetPathOfAssembly(Assembly.GetExecutingAssembly()),
+                    FileName = Process.GetCurrentProcess().MainModule.FileName,
                     Arguments = String.Format("--buildSolution {0}", pipeId),
                     WorkingDirectory = Environment.CurrentDirectory,
                     CreateNoWindow = false,

